.circleci/config.yml

```diff
@@ -384,7 +384,7 @@ jobs:
             make docker.all generate_yaml
       - run: bin/testEnvRootMinikube.sh wait
       - run: docker images
-      - run: make e2e_pilot HUB="${HUB}" TAG="${TAG}" TESTOPTS="--skip-cleanup -mixer=true -auth_enable=true -v1alpha3=true -v1alpha1=true -errorlogsdir=/home/circleci/logs -use-sidecar-injector=false --core-files-dir=/home/circleci/logs"
```

Comment:
this is not required already - runs only in periodic.

You want to change e2e-pilot job, and use "make test/minikube/noauth/e2e_pilot_alpha1"

Reply:
Done